jak-na-konflikty.md

```diff
@@ -0,0 +1,77 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
+**Cílem přípravy není mít co nejvíc argumentů k tomu, abys svůj protějšek ubil argumenty.** Cílem je schopnost nahlédnout na danou věc z více úhlů, vžít se do svého kolegy a pochopit jeho motivaci a chování.
+
+- Co je věcným "odemočněným" předmětem konfliktu? Jak se daný problém dá popsat co nejvíc věcně?
+- Jak bude předmět konfliktu vnímat a vidět tvůj kolega?
+- Jaké kolem toho máš emoce? Jak se cítíš?
+- Jaké kolem toho bude mít emoce tvůj kolega? 
+- Jaké asi byly motivace kolegy? Proč postupoval tak jak postupoval?
+
+## Před konfliktem
+
```

Comment:
Co třeba "Zeptej se na to někoho dalšího a prober to s ním, Jaký na to má názor?"

Reply:
Už tady by měla být zmínka o mediatorovi,

Reply:
@hauseja3 Když tady je obávám se přesně ten problém, že se člověk zeptá pár jinejch lidí a najednou buduje skupinu lidí, která někoho hejtuje.

Reply:
=> to je požadavek na ty další lidi... že musí udělat vše pro to, aby zůstali objektivní.

Reply:
A pokud objektivní být nedokážu, nebo cítí, že jejich objektivita není dostatečná, z konfliktu odstoupí a předá to radě starších. Jsem přesvědčenej, že my čtyři tohle dokážeme.

Reply:
@hauseja3 No já nevím, já jdu do konfliktu málokdy s tím, že jsem objektivní. Objektivní má být mediátor. Konfliktní strany jsou subjektivní a jejich (současná) objektivita je až výsledkem řešení toho konfliktu, ne?

Reply:
Asi jsem se blbě vyjádřil, psal jsem to na rychlo. Bavil jsem se o objektivitě mediátora. Mluvim tam o nich jako o "dalších lidech".

Požadavek na mediátora, aby zůstal objektivní. Pokud to nedokáže, musí být aspoň dostatečně sebereflexni, aby si to dokázal přiznat, odstoupil a passnul ten problém na řadu starších. Kde jak jsem už řekl jsem přesvědčený, že dokážeme objektivitu zajistit.

Reply:
@hauseja3 Tohle docela odpovídá bodu

* nehodnoť, pouze nabízej různé pohledy na situaci

v kapitole **Role mediátora**

Reply:
Asi bych dal větší důraz na to, že není tvoje chyba odstoupit a předat to dál.. o to mi jde

Reply:
@hauseja3 Přidal jsem to dolů tam kam to patří, tuhle konverzaci zavři, až si tohle přečteš 🙂